content/docs/source-git/_index.md

```diff
@@ -58,6 +60,16 @@ any way you want:
 * Script it yourself
 
 
+### How packit treats `git commit`s
+
+Packit distinguishes between 2 types of commits:
+1. Those that change code (usually pulled from upstream)
+2. Changes to packaging (spec file, other sources, changes to .packit.yaml)
+
+Packit treats code changes as downstream patches so it creates those files
```

Comment:
Not sure I understand....

Reply:
I dropped the paragraph and instead elaborated in the point 1